Bio/motifs/__init__.py

```diff
@@ -27,13 +27,17 @@ def parse(handle, format):
     """Parse an output file from a motif finding program.
 
     Currently supported formats (case is ignored):
-     - AlignAce:      AlignAce output file format
-     - MEME:          MEME output file motif
-     - MAST:          MAST output file motif
-     - TRANSFAC:      TRANSFAC database file format
-     - pfm:           JASPAR-style position-frequency matrix
-     - jaspar:        JASPAR-style multiple PFM format
-     - sites:         JASPAR-style sites file
+     - AlignAce:         AlignAce output file format
+     - ClusterBuster:    Cluster Buster position frequency matrix format
+     - XMS:              XMS matrix format
+     - MEME:             MEME output file motif
+     - MAST:             MAST output file motif
+     - TRANSFAC:         TRANSFAC database file format
+     - pfm_four_columns: Generic position-frequency matrix format with four columns. (cisbp, homer, hocomoco, neph, tiffin)
```

Comment:
Style wise, in ``Bio.SeqIO`` etc we've used the convention ``format-variant`` with a minus sign not an underscore, and generally one word variants. How about ``pfm-4column`` or ``pfm-4col`` here or even ``pfm-cols``? Likewise for the row-based pfm format (next line of the documentation).

In any case, looking at the code, both call the same pfm parser, so I would think just exposing a single ``pfm`` format ought to be fine and easier for the user.

Reply:
@peterjc **pfm-cols** and **pfm-rows** are fine for me. I never liked it that JASPAR uses **pfm** instead of **pfm-jaspar**. It would be nicer if at least the **pfm-jaspar** alias is added.
The **cols** and **rows** option is important for the parsing of the motifs.
Sometimes it could be autodetectable which one to use, but for a 4x4 motif matrix without ACGT or position annoation, this is impossible.

Reply:
Does that mean something is missing in the code to pass the row/col information to the parser?

Reply:
I missed there already was already ``pfm`` defined for JASPAR style PFM. Is this different from the 4 row / 4 col PFM?

If @mdehoon does not object, adding ``pfm-jaspar`` as an alias makes sense.

Reply:
The JASPAR style 'pfm' won't parse all of the other generic 4 row/column formats, just those from JASPAR. However, `pfm-rows` does handle the JASPAR style PFM, making the `pfm` command redundant and unnecessary really.

I'd suggest removing/deprecating the `pfm` command from the JASPAR module, as `pfm-rows` and `pfm-cols` will be much more flexible. If @mdehoon agrees, I will add a test or two for `pfm-rows` and `pfm-cols` as well.

Reply:
@j-andrews7 If I am not mistaken `pfm-rows` could also be used instead of most of the code in `_read_jaspar`. Only some code should be written to get the correct `identifier` (should be pfm `name`) and `name` (should be pfm `gene` or `name` if `gene` was not found) attributes: 
```
def _read_jaspar(handle):
    ...

        if head_match:
            identifier = head_match.group(1)
            if head_match.group(3):
                name = head_match.group(3)
            else:
                name = identifier

```

Reply:
@ghuls Yes, I also noticed that and you are correct that all that needs to be changed would be the identifier/name split. That function could also be deprecated with little work.

If @peterjc and/or @mdehoon thinks that's a good idea, I can also incorporate those changes.